src/style.css

```diff
@@ -1,3 +1,91 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
 body {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+}
+
+header {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.logo {
+  height: 60px;
+  display: flex;
+  align-items: center;
+  margin-left: 50px;
+}
+
+.logo img {
+  height: 40px;
   margin: 0;
 }
+
+nav {
+  display: flex;
+  align-items: center;
+  margin-right: 50px;
+}
+
+nav a {
+  height: 100%;
+  display: flex;
+  align-items: center;
+  color: #000;
+  text-decoration: none;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+}
+
+nav ul {
+  display: flex;
+  list-style: none;
+  align-items: center;
+  gap: 20px;
```

Comment:
The `gap` property is not supported in tests. Please use margins for spacing instead.